pkgs/applications/office/mytetra/default.nix

```
@@ -23,6 +23,8 @@ mkDerivation rec {
 
     substituteInPlace app/src/views/mainWindow/MainWindow.cpp \
       --replace ":/resource/pic/logo.svg" "$out/share/icons/hicolor/48x48/apps/mytetra.png"
+
+    sed -i 's/const char \*strcasestr/char *strcasestr/' thirdParty/mimetex/mimetex.c
```

Function provided by stdenv
```suggestion
    substituteInPlace thirdParty/mimetex/mimetex.c \
      --replace 'const char \*strcasestr' 'char *strcasestr'  
```

---

Done.